src/robot/world.ts

```diff
@@ -131,6 +134,25 @@ export function declareWorldClass(env: GlobalEnvironment): BuiltinClassVal {
         }
     );
 
+/** Example:
+Objekt kugel = welt.erzeugeKugel()
+
+wiederhole für i von 1 bis 100
+    kugel.setzeRadius(kugel.radius + i)
+ende
+*/
+
+    mkWorldMethod(
+        ENV.world.mth.CREATE_SPHERE,
+        (w, args) => {
+            if (args.length != 0)
+                throw new RuntimeError(ENV.world.mth.CREATE_SPHERE + `() erwartet keine Parameter!`);
+            let ret = instanceSphereObject([], env);
```

Comment:
```suggestion
            let ret = instanceSphereObject(args, env);
```
Error handling kann die Sphere Klasse selber machen?